internal_dev_guide.md

```diff
@@ -5,52 +5,48 @@ Internal dev guide (for AdaCore developers)
 
 ### Setup
 
-First step, which is one shot, is to setup a development environment. Follow
-these steps:
+1. Make sure you have a Python 3.7+ distribution (with `pip3`) available in
+   your environment. AdaCore's `e3-distrib` package is completely fine.
 
-1. Install dependencies via Anod
+2. Checkout LKQL
 
 ```sh
-anod install gnat
-anod install langkit_support -Qlalmaster
-anod install langkit -Qlalmaster
-anod install libadalang -Qlalmaster
+git clone git@github.com:AdaCore/langkit-query-language.git
+cd langkit-query-language
 ```
 
-2. Checkout LKQL
+3. Install Python dependencies via `pip`:
 
 ```sh
-git clone git@github.com:AdaCore/langkit-query-language.git
+pip3 install -r REQUIREMENTS.txt
```

Comment:
The requirements.txt file has not been added.

Reply:
And it would be nice smoke-testing it in GH CI

Reply:
nit: the `requirements.txt` is usually lowercase, which allows for eg `find`'ing it or autocomplete without too much thinking

Reply:
Oops, indeed, sorry! All should be fixed now.